Templates for videos

nrichers opened the pull request:
## Internal Notes for Reviewers

This PR adds a basic reveal.js presentation with slides you can use for different video intros:

- Videos based on training modules
- Videos that shows you how to perform a task
- FAQ-style videos that answer a question

## Example

<img width="1440" alt="image" src="https://github.com/user-attachments/assets/136b0cb4-d0b8-4396-af48-a2af86fae971">

<!--
PR instructions for release notes:

1. Pick at least one label:

- `internal` (skip Step 2, no release notes required)
- `highlight`
- `enhancement`
- `bug`
- `deprecation`
- `documentation`

2. In the next section, describe the changes so that an external user can understand them. Keep it simple and link to the docs with [Learn more ...](<relative-link>), if available.
-->

## External Release Notes

<!--- REPLACE THIS COMMENT WITH YOUR DESCRIPTION --->

nrichers commented:
@validbeck this PR is ready to be approved — I mention it in my standard for community videos that is part of Sprint 59 and should be merged.

validbeck reviewed (changes requested):
Very cool, the only thing is how does one render-preview/load the slides locally? I moved the slides into `site` on my local branch to see it, but is there a better way? Perhaps this could be included in the template instructions?

nrichers commented:
> Very cool, the only thing is how does one render-preview/load the slides locally? I moved the slides into `site` on my local branch to see it, but is there a better way? Perhaps this could be included in the template instructions?

This is really more of a "how do I use Quarto question?" but I added the instructions for how to render the presentation to https://www.notion.so/validmind/Community-Videos-7cdc46a3403940b4acd7a32df63d2f7a?pvs=4#10519127eb12801fa6dbcc7b34cab57d

(TL;DR: cd into the folder where the presentation lives and run `quarto preview index.qmd`.)
